plaidcloud/utilities/query.py

```diff
@@ -428,6 +425,30 @@ def bulk_insert_dataframe(self, table_object, df, append=False):
         """Pandas-flavored wrapper method to the SQLAlchemy bulk_save_objects
         bulk_insert_mappings(mapper, mappings, return_defaults=False, render_nulls=False)
         """
+        def df_to_csv_string(df, col_order):
+            # if six.PY3:
+            #     # convert any byte strings to string to avoid 'b' prefix bug in pandas
+            #     # https://github.com/pandas-dev/pandas/issues/9712
+            #     str_df = df.select_dtypes([np.object])
+            #     str_df = str_df.stack().str.decode('utf-8').unstack()
+            #     for col in str_df:
+            #         df[col] = str_df[col]
+            cs = StringIO()
+            cs.writelines(
+                df[col_order].to_csv(
+                    index=False,
+                    header=True,
+                    na_rep='NaN',
+                    sep='\t',
+                    encoding='UTF-8',
+                    quoting=csv.QUOTE_MINIMAL,
+                    escapechar='"'
+                )
+            )
+
+            # logger.info('----- CSV DATA DIRECTLY FROM PANDAS---- {}'.format(cs.getvalue()))
+            return cs.getvalue()
+
```

Comment:
I think this was deleted in a previous PR, somehow it wants to put it back. I guess that was a bad merge or something.

Reply:
Looks like it was the first commit in this PR that removed the use of six package. This code can likely be removed safely.